docs/getting-started/quick-start.md

```diff
@@ -182,6 +189,24 @@ Your final page should look similar to the one below.
 </html>
 ```
 
+## Installing CKEditor&nbsp;5 from a file
+
+If you do not want to build your project using npm, and you cannot rely on the CDN delivery, you can simply download CKEditor&nbsp;5 with all its plugins.
+
+1. <a href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/zip/ckeditor5-{@var ckeditor5-version}.zip">Download the .zip file with the latest CKEditor&nbsp;5 distribution.</a>
+2. Extract the .zip file into a dedicated directory inside your project. It is recommended to include the editor version in the directory name to ensure proper cache invalidation once a new version of CKEditor&nbsp;5 is installed.
+
+Files included in the .zip archive:
+
+* `index.html` &ndash; A sample with working editor.
+* `ckeditor5/ckeditor5.js` &ndash; The ready-to-use editor ESM bundle, containing the editor and all plugins. [Recommended build]
+* `ckeditor5/ckeditor.js.map` &ndash; The source map for the editor ESM bundle.
+* `ckeditor5/ckeditor5.umd.js` &ndash; The ready-to-use editor UMD bundle, containing the editor and all plugins. [Secondary build]
+* `ckeditor5/ckeditor5.umd.js.map` &ndash; The source map for the editor UMD bundle.
+* `ckeditor5/*.css` &ndash; The style sheets for the editor. You will use `ckeditor5.css` in most cases. Read about other files in the {@link getting-started/setup/css Editor and content styles} guide.
+* `translations/` &ndash; The editor UI translations (see the {@link getting-started/setup/ui-language Setting the UI language} guide).
+* The `README.md` and `LICENSE.md` files.
+
```

Comment:
I'm missing a small "what now" here. It only lists the files, and while it's mentioned at the very top that "add files to you project" maybe we should wrap it here, too?

Reply:
Good idea, what could be the closure? I thought for a second to add again a code block, but it would be almost the same as the CDN one. Also, in [the old docs](https://ckeditor.com/docs/ckeditor5/41.4.2/installation/getting-started/predefined-builds.html#zip-download) we didn't have the snippet.

Reply:
I wouldn't add any snippets, just something along the lines of "add these files to your project" or something like this?